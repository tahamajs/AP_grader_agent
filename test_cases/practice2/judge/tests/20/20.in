enter 1
reserve_table 1 1
enter 2
reserve_table 2
show_table_info 1
enter 3
reserve_table 3 2
reserve_table 4
enter 5
reserve_table 5
reserve_table 6 5
show_table_info 5
reserve_table 7
enter 8
reserve_table 8 2
switch 4
exit 1
show_table_info 1
reserve_table 9
enter 10
reserve_table 10 1
exit 2
show_table_info 2
reserve_table 11
reserve_table 12 3
exit 4
show_table_info 1
reserve_table 13
enter 14
reserve_table 14 1
reserve_table 15
exit 7
show_table_info 1
reserve_table 16
switch 10
reserve_table 17 2
enter 18
reserve_table 18
exit 10
show_table_info 2
