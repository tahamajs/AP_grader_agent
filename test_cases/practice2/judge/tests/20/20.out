Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Paria sits at table 1
Table 1: 3 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Amir Hossein sits at table 1
Table ID: 1
People at the table: Amir Hossein, Paria
Table remaining capacity: 2
Waiting queue length: 0
Table 1: 2 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Parsa sits at table 2
Mehrad sits at table 1
Table 1: 1 0
Table 2: 3 0
Table 5: 3 0
Table 3: 2 0
Table 6: 3 0
Table 4: 2 0
Fereshte sits at table 1
Amir Akbar sits at table 5
Table ID: 5
People at the table: Amir Akbar
Table remaining capacity: 2
Waiting queue length: 0
Mahdi enters the waiting queue of table 1
Table 1: 0 1
Table 2: 3 0
Table 5: 2 0
Table 3: 2 0
Table 6: 3 0
Table 4: 2 0
Taha sits at table 2
Mehrad switches seats with Taha!
Paria exits!
Table ID: 1
People at the table: Amir Hossein, Fereshte, Mahdi, Taha
Table remaining capacity: 0
Waiting queue length: 0
Sadegh enters the waiting queue of table 1
Table 2: 2 0
Table 1: 0 1
Table 6: 3 0
Table 4: 2 0
Table 5: 2 0
Table 3: 2 0
Hamid enters the waiting queue of table 1
Amir Hossein exits!
Table ID: 2
People at the table: Mehrad, Parsa
Table remaining capacity: 2
Waiting queue length: 0
Shaghayegh enters the waiting queue of table 1
Bob sits at table 3
Mehrad exits!
Table ID: 1
People at the table: Fereshte, Mahdi, Sadegh, Taha
Table remaining capacity: 0
Waiting queue length: 2
Ted sits at table 2
Table 1: 0 2
Table 2: 2 0
Table 5: 2 0
Table 3: 1 0
Table 6: 3 0
Table 4: 2 0
Babak enters the waiting queue of table 1
Erfan enters the waiting queue of table 1
Mahdi exits!
Table ID: 1
People at the table: Fereshte, Hamid, Sadegh, Taha
Table remaining capacity: 0
Waiting queue length: 3
Alex enters the waiting queue of table 1
Hamid switches seats with Parsa!
Sipamo sits at table 2
Table 2: 1 0
Table 6: 3 0
Table 1: 0 4
Table 5: 2 0
Table 4: 2 0
Table 3: 1 0
Amin sits at table 2
Hamid exits!
Table ID: 2
People at the table: Amin, Sipamo, Ted
Table remaining capacity: 1
Waiting queue length: 0
